elements/pf-popover/pf-popover.ts

```diff
@@ -393,6 +394,23 @@ export class PfPopover extends LitElement {
     this.#referenceTrigger?.removeEventListener('keydown', this.onKeydown);
   }
 
+  #getReferenceTrigger() {
+    const root = this.getRootNode() as Document | ShadowRoot;
+    return !this.trigger ? null : root.getElementById(this.trigger);
+  }
+
+
+  #triggerChanged() {
+    const oldReferenceTrigger = this.#referenceTrigger;
+    this.#referenceTrigger = this.#getReferenceTrigger();
+    if (oldReferenceTrigger !== this.#referenceTrigger) {
+      oldReferenceTrigger?.removeEventListener('click', this.show);
```

Comment:
@bennypowers this is my bad...should we use `toggle` here instead of `show`?

Reply:
yeah maybe? I'll take your word for it :grin: